libs/dal/src/repositories/integration/integration.entity.ts

```diff
@@ -19,6 +19,13 @@ export interface ICredentials {
   clientId?: string;
   projectName?: string;
   serviceAccount?: string;
+  applicationCode?: string;
+  defaultNotificationTitle?: string;
+  triggeredFunction?: any;
+  autoSubscribe?: boolean;
+  userId?: string;
+  tags?: object;
+  defaultNotificationImageURL?: string;
```

Comment:
@scopsy @davidsoderberg @ainouzgali @djabarovgeorge this entity is growing a lot. I would add it to technical debt to find a maintainable solution.